anta/tests/configuration.py

```diff
@@ -131,3 +131,42 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure("Following patterns were not found: " + ",".join(failure_msgs))
+
+
+class VerifyMcsClientMounts(AntaTest):
+    """Verifies the MCS Clients mount status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the MCS mount status on MCS Clients are mountStateMountComplete.
+    * Failure: The test will fail if the MCS mount status on MCS Clients are not mountStateMountComplete.
```

Comment:
I think this needs rewording as there are several failure tests and I am not sure what to expect if all clients are mounted or if only one is?

Reply:
The test will fail even if one of the MCS clients are not in mountStateMountComplete. Should I just add this ?